news/2024/2024-11-14-osu-world-cup-2024-round-of-16-recap.md

```diff
@@ -0,0 +1,218 @@
+
+Let's move onto the highlight matches of the week!
+
+### ::{ flag=RU }:: Russian Federation vs. ::{ flag=CN }:: China
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116133396)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2298021032)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Q3t9MojGc18?start=136&end=300" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Heading into this week, many a sharp eye was turned to the winners' bracket matchup between the Russian Federation and China. For Russia, it was an opportunity to reach the Quarterfinals for the fourth year in a row; meanwhile, China was looking to turn the tables after getting swept by Russia last year, and get past the round of 16.
+
+Both teams came out of the gates firing on all cylinders; China's first pick of [DT3](https://osu.ppy.sh/beatmapsets/1778687#osu/3642792) showcased one of the closest maps of the entire match, tightly contested all the way en route to a last-second China win by 42,537 points. Unfazed, Russia picked into their strength in the speed skill set, locking in [DT2](https://osu.ppy.sh/beatmapsets/49071#osu/154165) for a comfortable win. China backed up their DT3 win with a dominating performance on their second pick, [HD2](https://osu.ppy.sh/beatmapsets/1739433#osu/3555094), winning by over a million points, and making the score 1–2.
+
+The first break point would come from a surprising source: Russia picked the tapping-focused FM3 map, but they stumbled badly, something that China took advantage of as ::{ flag=CN }:: [lolol233](https://osu.ppy.sh/users/11375105) led the way to a win and a 3–1 lead. Their advantage would be short-lived, as China faltered on [NM2](https://osu.ppy.sh/beatmapsets/2274929#osu/4847509), followed by another massive Russian win on their own [HR2](https://osu.ppy.sh/beatmapsets/2274925#osu/4847498) pick to tie the match at 3 points apiece, leaving each team with one pick.
+
+China selected the remaining hidden map Escape for their last pick, but it backfired in the worst possible way, making Russia one point away from the win. For their final pick, Russia selected [DT1](https://osu.ppy.sh/beatmapsets/1126545#osu/2353845), and what ensued was a nail-biter. China held a narrow lead through much of the map, but into the ending Russia held a combo advantage as seconds counted down; a 200,000 point lead became 100,000, then 50,000, then 15,000 points as the final spinner appeared, but time ran out on Russia as China pulled off a break point to force the tiebreaker.
```

Comment:
China selected the remaining hidden map Escape for their last pick, but it backfired in the worst possible way as Russia won in dominant fashion to earn a match point at 4-3. For their final pick, Russia selected [DT1](https://osu.ppy.sh/beatmapsets/1126545#osu/2353845), and what ensued was a nail-biter. China held a narrow lead through much of the map, but into the ending Russia held a combo advantage as seconds counted down; a 200,000 point lead became 100,000, then 50,000, then 15,000 points as the final spinner appeared, but time ran out on Russia as China pulled off a break point to force the tiebreaker.